packages/inputs-minimal/src/textarea/index.jsx

```diff
@@ -76,14 +80,16 @@ const TextArea = ({
 
 TextArea.propTypes = {
   label: PropTypes.string,
-  placeholder: PropTypes.string,
+  name: PropTypes.string,
   error: PropTypes.string,
   value: PropTypes.string,
   onChange: PropTypes.func.isRequired,
   className: PropTypes.string,
   disableBorder: PropTypes.bool,
   maxLength: PropTypes.number,
```

Comment:
@inomn maybe we should make this a bit more generic - passing something like `hint`? The input length counter could be a HOC for both TextArea and TextField